Sources/StreamChatUI/ChatMessageList/MessageComposer/ChatMessageComposerView.swift

```diff
@@ -11,7 +11,7 @@ open class _ChatMessageComposerView<ExtraData: ExtraDataTypes>: _View, ThemeProv
     public private(set) lazy var container = ContainerStackView()
         .withoutAutoresizingMaskConstraints
 
-    public private(set) lazy var topContainer = ContainerStackView()
+    public private(set) lazy var topContainer = UIView()
         .withoutAutoresizingMaskConstraints
```

Comment:
Why this change?

Reply:
![image](https://user-images.githubusercontent.com/12814114/116394357-ba1cd580-a81a-11eb-9a7e-6f09e0be9d06.png)

Since there's no more left button on the top, it just easier to layout the views without the container. Or I would need to add an empty view to make the same layout. WDYT? Or is it the aim to always use the ContainerStackView? always?